apps/web/pages/event-types/[type].tsx

```diff
@@ -767,7 +767,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                         <label
                           htmlFor="location"
                           className="mt-2.5 flex text-sm font-medium text-neutral-700">
-                          <LocationMarkerIcon className="mt-0.5 h-4 w-4 text-neutral-500 ltr:mr-2 rtl:ml-2" />
+                          <LocationMarkerIcon className="mt-0.5 mb-4 h-4 w-4 text-neutral-500 ltr:mr-2 rtl:ml-2" />
```

Comment:
Since you are already modifying this view, I saw an improve in design that its near this change at line 747 to make clock icon centers with its text just right
`<ClockIcon className="mt-0.5 h-4 w-4 text-neutral-500 ltr:mr-2 rtl:ml-2" />{" "}
                              {t("duration")}`
 _to ->_
`<ClockIcon className="w-4 text-neutral-500 ltr:mr-2 rtl:ml-2" /> {t("duration")}`
Old:
![image](https://user-images.githubusercontent.com/4461358/156873706-952ad72f-7af4-4322-8fa2-89e2cc9056ad.png)
New: 
![image](https://user-images.githubusercontent.com/4461358/156873722-53b12eb6-6dfd-4754-8fda-aa4a85e60517.png)

Reply:
I don't see the difference 😅

Reply:
i see, its vertical aligned